style.css

```diff
@@ -8,15 +8,28 @@
             width: 100%;
         }
         
-        .menu {
-            width: 80%;
```

Comment:
Usunąłem width: 80%; i zamieniłem to na wygodną w użyciu klasę limit, która jest środkowana przez 
margin: 0 auto;